compiler/semantic/op.go

```diff
@@ -179,8 +178,6 @@ func semPool(ctx context.Context, scope *Scope, p *ast.Pool, ds *data.Source, he
 				Meta:      p.Spec.Meta,
 				Pool:      poolID,
 				Commit:    commitID,
-				ScanLower: lower,
-				ScanUpper: upper,
```

Comment:
Because this isn't be used any more. See https://github.com/brimdata/zed/pull/3930/files#diff-072c6f760a72cdd01fcb450006cda7ebdbf0ac0ff18baffff5db2c742b017ce2R88 range expressions get added to the pushdown filter.

Reply:
In that case, can/should we remove remove these field from compiler/ast/dag.CommitMeta?

Reply:
Indeed we should.

Reply:
Actually nevermind this removal is a mistake and should be added back.